Userland/DevTools/UserspaceEmulator/SoftFPU.cpp

```diff
@@ -107,6 +109,59 @@ long double SoftFPU::fpu_pop()
     return ret;
 }
 
+void SoftFPU::fpu_set_exception(FPU_Exception ex)
```

Comment:
Please squash these changes. There is no reason for this change to be separate from the initial FPU patch.